tests/test_springshare.py

```diff
@@ -0,0 +1,211 @@
+import transmogrifier.models as timdex
+from transmogrifier.helpers import parse_xml_records
+from transmogrifier.sources.springshare import SpringshareOaiDc
+
+SPRINGSHARE_FIXTURES_PREFIX = "tests/fixtures/oai_dc/springshare"
+
+LIBGUIDES_FIXTURES_PREFIX = "tests/fixtures/oai_dc/springshare/libguides"
```

Comment:
The 3 prefixes are bit duplicative, you could just use the `
SPRINGSHARE_FIXTURES_PREFIX` and add the the rest of the string for the other 2